lib/shared/units.js

```diff
@@ -50,33 +43,38 @@ const MEGABYTE_TO_BYTE_RATIO = 1e+6;
 const MILLISECONDS_IN_A_DAY = 86400000;
 
 /**
- * @summary Convert bytes to gigabytes
+ * @summary Convert bytes to megabytes
  * @function
  * @public
  *
  * @param {Number} bytes - bytes
- * @returns {Number} gigabytes
+ * @returns {Number} megabytes
  *
  * @example
- * const result = units.bytesToGigabytes(7801405440);
+ * const result = units.bytesToMegabytes(7801405440);
  */
-exports.bytesToGigabytes = (bytes) => {
-  return bytes / GIGABYTE_TO_BYTE_RATIO;
+exports.bytesToMegabytes = (bytes) => {
```

Comment:
We can't really control what unit it becomes so we'd have to parse and do some ugly stuff for this to work, I can't really think of a good, clean way to use the other function exclusively.